model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -356,6 +356,17 @@ case class MethodConfiguration(
   def toShort : MethodConfigurationShort = MethodConfigurationShort(name, rootEntityType, methodRepoMethod, namespace)
   def path( workspaceName: WorkspaceName ): String = workspaceName.path+s"/methodconfigs/${namespace}/${name}"
   def path( workspace: Workspace ): String = path(workspace.toWorkspaceName)
+
+  def addDefaultOutputs(): MethodConfiguration = {
```

Comment:
maybe call this withDefaultOutputs? Add sounds like a mutation which (rightly) this is not.